src/tomato/drivers/dummy/main.py

```diff
@@ -60,7 +60,10 @@ def get_status(
     logger.debug("in 'dummy.get_status'")
     dt = datetime.now(timezone.utc)
     metadata = {"address": address, "channel": channel}
-    ready = jobqueue.empty()
+    if jobqueue:
+        ready = jobqueue.empty()
+    else:  # this happens when called by driver_reset
+        raise NotImplementedError()  # FIXME
```

Comment:
I see. Since we're cancelling the job, we might as well do:

```suggestion
    else:  # this happens when called by driver_reset
        ready = False
```

Reply:
Actually I had to use `ready = True` in order for `ketchup cancel` to work properly.